docs/faststore/docs/customization/building-sections/override-use-cases/adding-an-image-to-the-alert-component.mdx

```diff
@@ -0,0 +1,154 @@
+    import Image from "next/image";
+    import { useMemo } from "react";
+
+    interface AlertWithImageProps
+        extends Omit<React.ComponentProps<typeof AlertSection>, "icon"> {
+        src: string;
+        alt: string;
+    }
+    ```
+
+   - This interface defines the props accepted by the `AlertWithImage` component.
+   - It inherits all props from the native Alert component, excluding the icon prop, which we intend to replace with an image.
+   - It adds a new prop named `src`, which specifies the image source URL.
+
+5. Create the `AlertWithImage` component:
+
+    ```tsx focus=10:31
+    import { useMemo } from "react";
+    import { AlertSection, getOverriddenSection } from "@faststore/core";
+    import { Image_unstable as Image } from "@faststore/core/experimental";
+
+    interface AlertWithImageProps
+        extends Omit<React.ComponentProps<typeof AlertSection>, "icon"> {
+        src: string;
+        alt: string;
+    }
+
+    export default function AlertWithImage(props: AlertWithImageProps) {
+        const { src, alt, ...otherProps } = props;
+
+        const OverriddenAlert = useMemo(
+            () =>
+            getOverriddenSection({
+                Section: AlertSection,
+                className: styles.alertWithImage,
+                components: {
+                    Icon: {
+                        Component: () => (
+                            <Image src={props.src} alt={props.alt} width={24} height={24} />
+                        ),
+                    },
+                },
+            }),
+        []
+    );
+
+    return <OverriddenAlert {...otherProps} icon="" />;
+    }
+    ```
+
+    - This function receives the `AlertWithImageProps`.
+    - It destructures the `src` prop containing the image source and remaining props using the spread operator (`...otherProps`).
+    - The core functionality lies within the `useMemo` hook.
+      - It defines the `OverriddenAlert` constant using `getOverriddenSection`.
+      - The override options specify:
+        - **`Section`:** The section to be overridden (Alert).
+        - **`components`:** An object containing the override for the Icon component.
+
+    - Inside the Icon override:
+      - **`Component`:** A function returning an `Image` component. This displays the image specified by the `src` prop.
+      - **`width`** and **`height`:** Optional props to set the size of the image.
+
+    - The use of `useMemo` ensures this override is only created once, improving performance.
+    - The function returns the `OverriddenAlert` component, spreading the `otherProps` and setting the `icon` prop to `noop` (effectively disabling the default icon behavior).
+
+    > ℹ️ For further details on code implementation, see the `AlertWithImage` folder available in the playground.store repository.
+
+6. Add the section to the Headless CMS by following the instructions available for sending components to the Headless CMS.
```

Comment:
which instructions?

Reply:
Instructions from the documentation [syncing components to the Headless CMS](https://developers.vtex.com/docs/guides/faststore/overrides-syncing-components-with-the-headless-cms). Just added the link there.